virt-install, virt-sparsify: add page (#4333)
tldr-pages/tldr: pages/common/virt-sparsify.md

# virt-sparsify

> > libguestfs tool to to make a virtual machine drive images thin-provisioned.
> > Use only for offline machines to avoid data corruption.
> > Home page: <https://libguestfs.org/>.

Reviewer: Same deal here as with the previous page. What about this?
```suggestion
> Make virtual machine drive images thin-provisioned.
> Use only for offline machines to avoid data corruption.
> More information: <https://libguestfs.org/>.
```

Author: I'd prefer to keep a libguestfs mention, cause it could help to match the utility with a distro package. But it is up to you.

> - Create sparsified comporessed image without snapshots from unsparsified:

Reviewer: Typo, and maybe this will read better :-)
```suggestion
- Create a sparsified compressed image without snapshots from an unsparsified image:
```

Author: Oh, indeed, thank you!

> `virt-sparsify --compress {{image.qcow2}} {{image_new.qcow2}}`

Reviewer: More consistency
```suggestion
`virt-sparsify --compress {{path/to/image.qcow2}} {{path/to/image_new.qcow2}}`
```

> - Sparsify image in place:

Reviewer: Perhaps this will read better?
```suggestion
- Sparsify an image in-place:
```

> `virt-sparsify --in-place {{/var/lib/libvirt/images/image.img}}`

Reviewer: Consistency again
```suggestion
`virt-sparsify --in-place {{path/to/image.img}}`
```
....unless images _have_ to be located in the directory you've specified there in order for it to work properly - in which case ignore this comment.
